.github/workflows/preview-env-check-regressions.yml

```diff
@@ -13,24 +17,56 @@ on:
                 required: false
                 default: harvester
 jobs:
+    configuration:
+        name: Configuration
+        runs-on: [self-hosted]
+        outputs:
+            skip: ${{ steps.configuration.outputs.skip }}
+            name: ${{ steps.configuration.outputs.name }}
+            version: ${{ steps.configuration.outputs.version }}
+            infrastructure_provider: ${{ steps.configuration.outputs.infrastructure_provider }}
+        steps:
+            - name: "Set outputs"
+              id: configuration
+              run: |
+                  if [[ '${{ github.event.inputs.name }}' != '' ]]; then
+                      # The workflow was triggered by workflow_dispatch
+                      {
+                          echo "version=${{ github.event.inputs.version }}"
+                          echo "name=${{ github.event.inputs.name }}"
+                          echo "infrastructure_provider=${{ github.event.inputs.infrastructure_provider }}"
+                          echo "skip=false"
+                      } >> $GITHUB_OUTPUT
+                  else
+                      # The workflow was triggered by workflow_run
+                      {
+                          echo "version=main-gha.${{ github.event.workflow_run.run_number }}"
```

Comment:
I couldn't find any information about what it inside of the workflow_run payload - [their docs](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_run) just says the key exists and is required, but not what it contains.